apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/DateTimestampReadTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright 2021 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests
+{
+    [Collection(nameof(DateTimestampTableFixture))]
+    [CommonTestDiagnostics]
+    public class DateTimestampReadTests
+    {
+        private readonly DateTimestampTableFixture _fixture;
+
+        public DateTimestampReadTests(DateTimestampTableFixture fixture) =>
+            _fixture = fixture;
+
+        public static IEnumerable<object[]> TestDates =>
```

Comment:
I think this can be made simpler with `TheoryData<DateTime>` but I can do that in a follow-up PR. (This is much better than before already.)